bio_hansel/kmer_count/utils.py

```diff
@@ -54,17 +54,19 @@ def error_rate(hist: pd.DataFrame) -> float:
     :return:
             float value containing the error rate.
     """
-    n_singleton_kmers = float(hist[hist.freq == 2].obs)
+    n_singleton_kmers = float(hist[hist.freq == 1].obs)
     logging.debug('n_singleton_kmers %s', n_singleton_kmers)
     # get the number of unique k-mers within the observation
     #   We can simply follow the "curve of the graph" to find the number of unique k-mers.
-    n_unique_kmers = hist.obs.sum()
+    n_unique_kmers = (hist['freq'] * hist['obs']).sum()
```

Comment:
Would this actually give the number of unique kmers rather than just the total number of kmers?

Reply:
Yeah, verified this with Marinier and now we're getting the value we expect for unique k-mers